content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,63 @@
+
+## How It Works
+
+Contrast's data masking feature protects sensitive data in your applications by preventing the Contrast agent from sending it to external systems: the Contrast UI, syslog or the security log. Data masking can also protect information by preventing the agent from sending it as part of Contrast vulnerability or attack reports. 
+
+Sensitive data masking is part of policy management by default for organizations with at least one active Assess or Protect license. Contrast offers several categories of sensitive data, or data types, that are comprised of specific keywords that the agent automatically identifies and redacts in reports. 
+
+### Agent identification and masking
+
+Contrast agents mask sensitive data in query parameters, request headers, cookies and forms. Your agent identifies sensitive data by searching for specific keywords used in the input name. If the agent finds a match, it redacts the value for that input, and replaces it with a placeholder with the format `contrast-redacted-{datatype}`, where `datatype` is the category of sensitive data to which the keyword belongs.
+
+Contrast agents do **not** mask individual fields in request bodies with a content type other than `application/x-www-form-urlencoded`; however, you can configure the agent to mask the entire request body. Contrast agents also do not mask data that appears in the data flow portion of a vulnerability report, if using Assess, or in the vector of an attack event, if using Protect. 
+
+> **Note:** Contrast agents make a “best effort” attempt to avoid printing sensitive data in debug log statements; however, it’s possible that sensitive data could appear in the Contrast log, if the log level is set to DEBUG or lower. Whenever possible, you should avoid setting production systems to log at DEBUG or lower. If a system that deals with sensitive data is set to log at DEBUG or lower, you should take steps to ensure that those logs are not being sent to an external system to avoid leaking any sensitive data. 
+
+#### Example
+
+The following sample HTTP request sent by an agent as part of a vulnerability report shows two inputs that the agent identified as sensitive as well as the placeholders it used to mask the values of the input before sending the report to the Contrast UI, syslog server or security log. 
+ 
+```
+POST/employee/5 HTTP 1.1 
+
+Host: example.com 
+
+Content-Type: application/x-www-form-urlencoded 
+
+Content-Length: 30 
+
+Authorization: contrast-redacted-authentication 
+ 
+
+ssn=contrast-redacted-id&department=sales 
+
+``` 
+
+In this case, the header value is masked because "Authorization" matches a keyword in the "Authentication" data type, and the form parameter is redacted because "ssn" matches a keyword in the "ID" data type for your Contrast organization. (Keyword matches are case insensitive.) <!-- TODO: are the above placeholder data types and keywords actual real types/keywords for this feature?  -->
+
```

Comment:
Can someone confirm that the placeholders are correct?

Reply:
this looks ok to me but mark is a better authority on the UI.  If he doesn't get to this, it all reads correct to me.

Reply:
`ssn` is a default keyword and the group id would be shown as `contrast-redacted-government-id`
`Authorization` is not currently a default - although it probably should be.  Do you want another example to use instead?

Reply:
Changed to `apikey`, which would be shown as `contrast-redacted-authentication-info`, as discussed

Reply:
This should say 

" "ssn" matches a keyword in the "Government ID" data type" 
and
" "apikey" matches a keyword in the "Authentication Info" data type"